### DISCUSSION TITLE
SX1276/77/78/79

### DISCUSSION THREAD

**Discussion by DavidMJones6352** [merbanan/rtl_433#2149] [General]
Has anyone played with the SX1276 that's on the esp32 Lora boards? 
I'm hoping to receive data from various Ambient sensors including the WH31E temp/humidity sensor.

**Comment by matthias-bs** [comment]
Maybe this helps:
https://github.com/matthias-bs/BresserWeatherSensorReceiver
I am currently working on an update which allows to collect data from multiple radio messages.
HopeRF95W is compatible to SX1276. The transceiver is connected to the ESP32 via SPI. Decoding of FSK signals works. OOK should work, too.

**Comment by serialrf433** [comment]
Its implemented now.
This to build something on your own: https://github.com/NorthernMan54/rtl_433_ESP
Or use the full blown functionality with sending the data (in your case WH31E sensor) over MQTT to a server of your choice: https://github.com/1technophile/OpenMQTTGateway
